File: Projekat/src/app/models/Schedule.ts
import { Line } from './Line';
import { ScheduleType } from './ScheduleType';

export class Schedule{
    Id: number;
    Departure: string;
    DayInWeek: string;
    ScheduleTypeId:number;
    ScheduleType : ScheduleType
    LineId:number;
    Line:Line;
    
   
    constructor(private id: number, private departure: string, private lineid:number,private day:string,private schtid:number,private schtype : ScheduleType,private line:Line) { 
        this.Id=id;
        this.Departure=departure;
        this.LineId=lineid;
        this.DayInWeek=day;
        this.ScheduleTypeId=schtid;
        this.ScheduleType=schtype;
        this.Line=line;
      }


}

File: WebApp/Models/Schedule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class Schedule
    {
        public int Id { get; set; }
        [Required]
        public string Departure { get; set; }
        [Required]
        public string DayInWeek { get; set; } 
        public int ScheduleTypeId { get; set; }
        //public ScheduleType ScheduleType { get; set; }
        public int LineId { get; set; }
        public Line Line { get; set; }


        //public Schedule(int id,string dep,int lineid)
        //{
        //    Id = id;
        //    Departure = dep;
        //    LineId = lineid;
        //}
    }
}